store/imagestore/store.go

```diff
@@ -278,6 +278,9 @@ func (s *Store) Close() error {
 
 // backupDB backs up current database.
 func (s *Store) backupDB() error {
+	if os.Geteuid() != 0 {
+		return errors.New("database schema updated, re-run as root to perform the update")
```

Comment:
Two nits here:
 1. as we bubble-up this error, can we move it to its own package variable?
 2. I'm dumb, and the wording initially confused me about the status of the update at this point. I'd explicitly say `database schema needs to be updated, re-run...`.